redwood-layout-shared-test/src/commonMain/kotlin/app/cash/redwood/layout/utils.common.kt

```diff
@@ -26,7 +37,7 @@ const val Red: Int = 0xffff0000.toInt()
 const val Green: Int = 0xff00ff00.toInt()
 const val Blue: Int = 0xff0000ff.toInt()
 
-fun Color(
+fun argb(
```

Comment:
The widget factory function is named `Color` and I was getting confused. Figured I'd appease ktlint while I was at it.